src/app.js

```diff
@@ -1 +1,39 @@
 'use strict';
+/* eslint-disable no-console */
+
+const { terminal } = require('./terminal');
+const { generateNumber } = require('./generateNumber');
+const { countBullsAndCows } = require('./countBullsAndCows');
+const {
+  WIN_MESSAGE,
+  INVALID_INPUT_MESSAGE,
+  NUMBER_MESSAGE,
+} = require('./constants');
+
+const randomNumber = generateNumber();
+let userNumber;
+
+function bullsAndCows() {
+  terminal.question(NUMBER_MESSAGE, (userInput) => {
+    userNumber = userInput;
+
+    if (userNumber === randomNumber) {
+      console.log(WIN_MESSAGE);
+      terminal.close();
+
+      return;
+    }
+
+    if (typeof +userInput !== 'number' || userInput.length !== 4) {
+      console.log(INVALID_INPUT_MESSAGE);
+      bullsAndCows();
```

Comment:
According to the task you are supposed to chek if user entered different numbers -"the program check the entered number (if it has exactly 4 different digits)"